src/pages/climbs/[id].tsx

```diff
@@ -440,7 +440,7 @@ const ClimbData: React.FC<{
 
         {!editMode && (
           <div className='mt-8'>
-            <TickButton climbId={climbId} name={name} grade={gradesObj.toString()} />
+            <TickButton climbId={climbId} name={name} grade={gradesObj.toString()} climbType={{}} />
```

Comment:
Maybe I should just remove this tick buttons, since this is the "old" page....

Reply:
you can leave the button there until we delete the page